lib/src/util/dom_util.dart

```diff
@@ -149,3 +149,27 @@ void setSelectionRange(/* TextInputElement | TextAreaElement */Element input, in
     throw new ArgumentError.value(input, 'input', 'must be an instance of `TextInputElementBase`, `NumberInputElement` or `TextAreaElement`');
   }
 }
+
+/// Custom implementation to avoid issues with `selectionStart` when accessing
```

Comment:
See [the Dartlang style guide here](https://www.dartlang.org/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs). This comment would be more compliant as something like:


> Returns the index of the first selected character in [input].
>
>  To workaround [this chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=324360), returns `null` for [EmailInputElement] or [NumberInputElement] on Chrome. `null` will also be returned for any [input] that does not support `selectionStart`.